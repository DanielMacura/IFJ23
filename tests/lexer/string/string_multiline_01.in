let message = """
Hello,
    
    How are you doing today?
    
Best regards,
    John
    """