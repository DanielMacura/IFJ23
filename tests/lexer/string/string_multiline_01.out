KW_LET
VARIABLE
EQUALS
STRING:Hello,\010\032\032\032\032\010\032\032\032\032How\032are\032you\032doing\032today?\010\032\032\032\032\010Best\032regards,\010\032\032\032\032John\010\032\032\032\032
EOF
